server/tilesource/test.py

```diff
@@ -84,5 +142,5 @@ def getTile(self, x, y, z):
         )
 
         output = StringIO()
-        image.save(output, 'JPEG')
+        image.save(output, 'PNG')
```

Comment:
Real tiles are almost always JPEG-encoded in photographic images. I'm fine with PNG as a default for TestTileSource, but let's make it a parameter (something like ``encoding='png'``), so we can use JPEG if necessary.